testsuite/shared/src/main/java/org/jboss/as/test/integration/security/common/SecurityTraceLoggingServerSetupTask.java

```diff
@@ -37,6 +37,7 @@
     @Override
     protected Collection<String> getCategories(ManagementClient managementClient, String containerId) {
         return Arrays.asList("org.jboss.security", "org.jboss.as.security", "org.picketbox",
-                "org.apache.catalina.authenticator", "org.jboss.as.web.security", "org.jboss.as.domain.management.security");
+                "org.apache.catalina.authenticator", "org.jboss.as.web.security", "org.jboss.as.domain.management.security",
```

Comment:
we could probably remove "org.apache.catalina.authenticator", "org.jboss.as.web.security" as they are no longer in use since we moved from jbossweb --> undertow